docs/app/documentation/component-docs/localization-editor/localization-editor-docs.component.html

```diff
@@ -0,0 +1,56 @@
+<h2>Localization Editor</h2>
+
+<description>
+    Basic usage of Localization Editor, to add some text on add-on there is <code>[label]</code> input.
+</description>
+<component-example [name]="'ex1'">
+  <p>
```

Comment:
Remove <p> tags, I know they're present in other components and they should be removed there too, eventually